ape_aws/kms/_cli.py

```diff
@@ -93,22 +88,20 @@ def create_key(
     metavar="str",
 )
 @click.argument("alias_name")
+@click.argument("private_key")
```

Comment:
This will ensure that it is a proper path (to an encrypted keyfile using `eth-keyfile`), but also probably want to make it optional and then if a path is not given ask for a private key to be input directly
```suggestion
@click.argument("private_key", type=click.Path)
```

Reply:
I turned this into an option because of the mnemonic, but we can change that around as well. If it doesn't exist, we can prompt the user for either a mnemonic, if they don't want to do that, they can give us a private key hex. Or I can leave the interface as is. Let me know